virtual-voicebot-backend/script/whisper_server2.py

```
@@ -297,22 +295,43 @@ async def transcribe_stream(websocket: WebSocket):
-            await websocket.close()
+                try:
+                    await websocket.close()
+                except Exception:
+                    logger.warning("transcribe_stream failed to close websocket", exc_info=True)
             return
     except WebSocketDisconnect:
         return
 
 if __name__ == "__main__":
-    uvicorn.run(app, host="0.0.0.0", port=9010)
+    host = os.environ.get("HOST", "0.0.0.0")
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)